sdk/cognitivelanguage/ai-language-text/CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0-beta.1 (Unreleased)
 
 ### Features Added
 
+- Supports service version 2022-10-01-preview by default instead of 2022-05-01.
+- Adds back support for extractive summarization and FHIR.
```

Comment:
"Adds back" assumes they've read the entire changelog up till now. Doesn't seem important. Probably just "Adds support for" like all the others.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/23208/commits/538c4c6499ae22ebebf39ae2ad4fb21df682592f